pkg/virt-controller/services/template.go

```diff
@@ -1413,6 +1413,12 @@ func getMemoryOverhead(vmi *v1.VirtualMachineInstance) *resource.Quantity {
 		overhead.Add(resource.MustParse("16Mi"))
 	}
 
+	// When use uefi boot on aarch64 with edk2 package, qemu will create 2 pflash(64Mi each, 128Mi in total)
+	// it should be considered for memory overhead
+	if runtime.GOARCH == "arm64" {
```

Comment:
Hi, Roman, I have some confuse about implication here. Unit tests are supposed to test public function rather than private function. We need to pass the architecture information to a public function, thus we can easily test the function for different architecture. 
For code here, I can see two way to introduce architecture to public function:
1. add a new architecture variable to stuct `templateService`, then initialize the architecture information in function `NewTemplateService`
```
type templateService struct {
        ...
        launcherSubGid             int64
        cpuArch                         string
}

func NewTemplateService(launcherImage string,
                        ...
                        cpuArch string)
```

2. add a parameter to public `func (t *templateService) RenderLaunchManifestNoVm(vmi *v1.VirtualMachineInstance, arch string)` and `func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance, arch string)`

Do you have any suggestion?

Reply:
1. sounds right. No need to pass it  to every function.